pom.xml

```diff
@@ -60,31 +60,31 @@
       </extension>
     </extensions>
     <plugins>
-			<plugin>
-				<groupId>org.apache.maven.plugins</groupId>
-				<artifactId>maven-scm-publish-plugin</artifactId>
-				<version>3.0.0</version>
-				<configuration>
-				<scmBranch>gh-pages</scmBranch>
-				<username>${env.GHPAT_USER}</username>
-				<password>${env.GHPAT_TOKEN}</password>
-				</configuration>
-			</plugin>
-			<plugin>
-				<groupId>org.apache.maven.plugins</groupId>
-				<artifactId>maven-site-plugin</artifactId>
-				<version>3.8.2</version>
-				<dependencies>
-					<dependency>
-						<groupId>org.apache.maven.doxia</groupId>
-						<artifactId>doxia-module-markdown</artifactId>
-						<version>1.6</version>
-					</dependency>
-				</dependencies>
-				<configuration>
-					<relativizeDecorationLinks>false</relativizeDecorationLinks>
-				</configuration>
-			</plugin>
+	<plugin>
+		<groupId>org.apache.maven.plugins</groupId>
+		<artifactId>maven-scm-publish-plugin</artifactId>
+		<version>3.0.0</version>
+		<configuration>
+			<serverId>github</serverId>
```

Comment:
I'm not absolutely certain about this ... but I'm hopeful, and it's neater for the moment.

Reply:
My hopes are dashed. Back to user/pass,.